etna/lib/etna/application.rb

```diff
@@ -84,26 +88,13 @@ def find_descendents(klass)
     end
   end
 
-  def run_command(config, cmd = :help, *args)
-    cmd = cmd.to_sym
-    if commands.key?(cmd)
-      commands[cmd].setup(config)
-      commands[cmd].execute(*commands[cmd].fill_in_missing_params(args))
-    else
-      commands[:help].execute
-    end
+  def run_command(config, *args)
+    cmd, cmd_args = find_command(*args)
```

Comment:
find_command is the new interface for routing command line arguments to the correct inner command.